stdlib/public/Concurrency/Clock.cpp

```diff
@@ -159,7 +153,7 @@ switch (clock_id) {
       *nanoseconds = suspending.tv_nsec;
 #elif defined(_WIN32)
       *seconds = 0;
-      *nanoseconds = 1000;
+      *nanoseconds = 100;
```

Comment:
`QueryUnbiasedInterruptTimePrecise()` has a unit of 100ns, so its highest possible resolution is 100ns. Neat.